coss/club/templates/pages/index.html

```diff
@@ -12,52 +12,83 @@
     <h3>
       {{ page.homepage.header_subheading }}
     </h3>
-    <p class="lead">
+    <div class="lead col-sm-6">
       {{ page.homepage.header_text|safe }}
-    </p>
-    <p class="lead">
-      <a href="{{ page.homepage.header_link }}">Signup Now!</a>
-    </p>
+    </div>
+    <div class="lead">
+      <a href="{{ page.homepage.header_link }}" class="btn btn-outline-primary">Want to join our newsletter?</a>
+    </div>
   </div>
 {% endblock %}
 
 {% block main %}
-  <h3>Types of Clubs</h3>
-
-  <div class="row club-types">
+  <div class="row">
     <div class="col-md-6">
-      <img class="img-responsive" src="/static/img/opensource.jpg" alt="">
-
-      <h4>Open Source</h4>
+      <h4>Features Groups</h4>
+    </div>
+    <div class="col-md-6 text-right">
+      <a class="btn btn-outline-primary">Register a Group</a>
+    </div>
+  </div>
 
-      <div>
-        Students with a passion for technology working together to build the open web
-        of University and College campuses.
+  <div class="row group-types">
+    <div class="col-sm-4">
+      <div href="#" class="card">
+        <h4 class="card-header">Open Source <img class="float-right mt-2" src="/static/img/right-arrow-black.svg" alt="more"></h4>
+        <img class="img-fluid" src="/static/img/top-1.jpg" alt="Card image cap">
+        <div class="card-block">
+          <p class="card-text">
+            Students with a passion for technology working together to build the open web
```

Comment:
Same here, let's use a field with a `blank=True` attr everywhere that we have hard-coded text.